examples/SerialIface/SerialIface.ino

```diff
@@ -85,21 +95,36 @@ void processBinaryCmds() {
   }
 }
 
+void processFastBinaryCmds() {
+  while (Serial.available() >= FAST_BINARY_CMD_SIZE) {
+    byte cmd[FAST_BINARY_CMD_SIZE];
+    Serial.readBytes(cmd, FAST_BINARY_CMD_SIZE);
+
+    if (RESET_CMD != cmd[0]) {
+      opl2.write(cmd[0], cmd[1]);
+    } else {
+      opl2.init();
+    }
+    Serial.write(ACK_RSP);
+  }
+}
+
 void waitForBufSizeCmd() {
   if (Serial.available() >= 5) {
     String cmd = Serial.readString();
-    if (cmd.equals(BUF_SIZE_CMD)) {
-      ready = true;
-      Serial.println(SERIAL_RX_BUFFER_SIZE);
+    if (cmd.equals(BUF_SIZE_CMD) || cmd.equals(FAST_BUF_SIZE_CMD)) {
+      if (cmd.equals(FAST_BUF_SIZE_CMD)) {
+        processCmdFn = &processFastBinaryCmds;
+      } else {
+        processCmdFn = &processBinaryCmds;
```

Comment:
Not a biggy, but once you've initialized with a fast or regular buffer command you're stuck with it until you reset the Arduino.

Reply:
Good point, it would be nice if there was a way to gracefully exist each protocol.

Reply:
I don't see this as a big problem, since the Arduino will reset when reopening the serial port.